dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/relationship/RelationshipRequestParamsMapper.java

```diff
@@ -28,56 +28,73 @@
 package org.hisp.dhis.webapi.controller.tracker.export.relationship;
 
 import static org.apache.commons.lang3.BooleanUtils.toBooleanDefaultIfNull;
+import static org.hisp.dhis.tracker.imports.TrackerType.ENROLLMENT;
+import static org.hisp.dhis.tracker.imports.TrackerType.EVENT;
+import static org.hisp.dhis.tracker.imports.TrackerType.TRACKED_ENTITY;
 import static org.hisp.dhis.webapi.controller.tracker.export.RequestParamUtils.validateDeprecatedParameter;
 
+import java.util.Objects;
+import java.util.stream.Stream;
 import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.ObjectUtils;
 import org.hisp.dhis.feedback.BadRequestException;
 import org.hisp.dhis.tracker.export.relationship.RelationshipOperationParams;
+import org.hisp.dhis.tracker.imports.TrackerType;
 import org.hisp.dhis.webapi.common.UID;
 import org.springframework.stereotype.Component;
 
 /**
- * Maps query parameters from {@link RelationshipsExportController} stored in {@link
- * LegacyRequestParams} to {@link RelationshipOperationParams} which is used to fetch events from
- * the DB.
+ * Maps operation parameters from {@link RelationshipsExportController} stored in {@link
+ * RequestParams} to {@link RelationshipOperationParams} which is used to fetch relationships from
+ * the service.
  */
 @Component
 @RequiredArgsConstructor
 class RelationshipRequestParamsMapper {
 
+  private static boolean hasMoreThanOneParamNotNull(UID trackedEntity, UID enrollment, UID event) {
```

Comment:
Since its not related to params we can make it more generic like 

```suggestion
  private static boolean hasMoreThanOneNonNull(Object... values) {
```

I would put this function to the bottom as `map` should be the highlight of this class 😁

Reply:
Formatter is moving static methods at the top of the class.